monitoring/monitorlib/messagesigning/message_signer.py

```diff
@@ -0,0 +1,117 @@
+import base64
+import time
+from Crypto.PublicKey import RSA
+from Crypto.Signature import pkcs1_15
+from Crypto.Hash import SHA256
+from loguru import logger
+import urllib.parse
+from monitoring.monitorlib.messagesigning.hasher import get_content_digest
+
+
+def get_x_utm_jws_header(cert_url):
```

Comment:
extracted this function into `uft.py` since its not defined in IETF. Okay to keep the name as is? Everything else in this file should be in IETF.

Reply:
My personal preference would be to name it something like ietf.py to better indicate its content scope (to me, message_signer.py implies I should expect to find a class named MessageSigner in the module), but message_signer.py isn't unreasonable to contain generic/standard message signing tools.  I'd also personally prefer to name the action group rather than the actor as we don't have any other files I'm thinking of that name the actor (e.g., we have testing.py rather than tester.py, versioning.py rather than versioner.py or version_checker.py, auth.py rather than authorizer.py, fetch rather than fetcher, inspection.py rather than inspector.py, etc)